test/moduleTestCases/urls/expected.css

```diff
@@ -2,7 +2,7 @@
 	background: url({./module});
 	background: url({./module});
 	background: url("{./module module}");
-	background: url('{./module module}');
+	background: url("{./module module}");
```

Comment:
Why we change `'` to `"`? Can we save original quotes?

Reply:
In principle it could be done. It will complicate things quite a bit, though, since the whole point of the update is to quote based on module-content instead of name. The type of quotes would have to be passed to the client (something like `escapeSingle(require(<resource>))`).

I skipped it, since I can't really imagine a scenario where it would be important to use single quotes. `svg-url-loader` already assumes double-quotes, since this is what is commonly used in HTML, I assume other loaders will make similar choices. If not, the worst case scenario is a slightly longer string.

If this is implemented, there should maybe also be an `escapeUnquoted` function for handling unquoted modules. That way the quotes used will be entirely up to the user, the loader will just ensure valid CSS is produced.